### ISSUE TITLE
Heirloom problem 

### ISSUE THREAD

**Issue by Onirion** [dalaranwow/dalaran-wow#4968]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**:  i sent "Champion Herod's Shoulder" from my Blood Elf Hunter (Onirion) to my Tauren Druid (Tamenchia) and the item type remains in Mail, and i can't wear it.

**Expected behaviour**: "Champion Herod's Shoulder" should be a leather item until level 40 and has to change his type of armor accordingly with the character that is wearing it. 

**Steps to reproduce the problem**:

1. Send "Champion Herod's Shoulder" from a hunter to a druid
2. See if the item don't change his tipe to leather
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

Champion Herod'
![champion herod s shoulder bug](https://user-images.githubusercontent.com/33135754/32076564-aea7321a-baa0-11e7-8393-1e0a2bf85d6c.jpg)
s Shoulder: http://www.wowhead.com/item=42950/champion-herods-shoulder

**Include Screenshots from the issue if necessary**

**Comment by Dibbarra** [comment]
That only works for classes that can upgrade to wear mail. So only hunters and shamans can equip those shoulders. its working as intended.

**Comment by Onirion** [comment]
Ok thank you very much for the explanation :) i didn't knew this detail
